### ISSUE TITLE
Depreciate {width}{heihgt}on BBC image resized

### ISSUE THREAD

**Issue by Spuds** [elkarte/Elkarte#1935]
http://www.elkarte.net/community/index.php?topic=2364

**Comment by emanuele45** [comment]
Tracked here for completeness:
Re-reading this, and looking to "take action", I realised that:
1. starting with 1.0.1 the bbcode uses max-width and max-height, so not a *huge* problem,
2. removing this option is not "just" remove the bbcode, because if this option is set, Elk fetches the image while posting the message and verifies the sizes are smaller than the allowed.

So, we have several options: first, since we do not really care about the size of the image any more (because we are using max-*), we can just remove the fetching of the image reducing posting time. This irrespective of anything else.
Then, we can either decide that the image size is defined by the theme and remove the settings altogether, or keep the option in case admins want to have smaller images.

Opinions?

ETA: and let's not forget that width and height set with the BBC are not only for resizing, but are to set a size, in the sense that I can decide to post a certain width/height (unless it's bigger than the max set in the admin panel/css).

In other words: another mess of options! xD

**Comment by emanuele45** [comment]
@Spuds do you think this can be fixed or should be postponed for more investigations?

**Comment by Spuds** [comment]
At this lateness in 1.03 we may want to hold off for a 1.04 (assuming) We would have to do it early on thought since I think it will require several test cases.  I think its all safe, but I'm guessing there will be some unintended consequences as well..

**Comment by emanuele45** [comment]
Yups, that's what I thought me too.
Moving to 1.0.4.
